Source/Engines/ModuleEnginesRF.cs

```diff
@@ -685,7 +685,7 @@ public override void UpdateSolver(EngineThermodynamics ambientTherm, double alti
                 {
                     if (EngineIgnited && ignited && throttledUp && rfSolver.GetRunning())
                     {
-                        double state = ullageSet.GetUllageStability();
+                        double state = ullageSet.GetUllageProbability();
```

Comment:
I'd shove the Math.Pow down into GetUllageProbability because otherwise the probability is a lie.

Reply:
Yeah went out for a jog after this and that's what I figured I should do.  Tomorrow though.

Reply:
👍